### PR TITLE
CDAP-3094 Add HBase 1.0 to installation guide

### PR DESCRIPTION
**Pull request by ghelmling**

### CODE REVIEW

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 1]
```diff
@@ -634,13 +634,15 @@ recompiled and re-deployed.)
    - Using Yum (on one line)::
 
        $ sudo yum install cdap cdap-gateway \
-             cdap-hbase-compat-0.94 cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
+             cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
+             cdap-hbase-compat-1.0 cdap-hbase-compat-1.0-cdh \
```
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: So, ``cdap-hbase-compat-1.0-cdh`` gets installed even if you are running on HDP?

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 2]
```diff
@@ -146,29 +146,29 @@ Hadoop/HBase Environment
...
-| **Hive**      | Apache            | Version 0.12.0 through 0.13.1                    |
-+               +-------------------+--------------------------------------------------+
-|               | CDH or HDP        | (CDH) 4.3.x through 5.3.3 or (HDP) 2.0 or 2.1    |
-+---------------+-------------------+--------------------------------------------------+
++---------------+-------------------+----------------------------------------------------+
+| Component     | Source            | Required Version                                   |
++===============+===================+====================================================+
+| **HDFS**      | Apache Hadoop     | 2.0.2-alpha through 2.5.0                          |
++               +-------------------+----------------------------------------------------+
+|               | CDH or HDP        | (CDH) 5.0.0 through 5.4.4 or (HDP) 2.0, 2.1 or 2.2 |
```
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: (HDP) 2.0, 2.1, or 2.2 
[I hate to ask, as I know adding this comma is a pain to change.]

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 3]
```diff
@@ -634,13 +634,15 @@ recompiled and re-deployed.)
...
+             cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
+             cdap-hbase-compat-1.0 cdap-hbase-compat-1.0-cdh \
              cdap-kafka cdap-master cdap-security cdap-ui
 
    - Using APT (on one line)::
 
        $ sudo apt-get install cdap cdap-gateway \
-             cdap-hbase-compat-0.94 cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
+             cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
+             cdap-hbase-compat-1.0 cdap-hbase-compat-1.0-cdh \
```
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: ibid.

File: cdap-docs/admin-manual/source/installation/installation.rst (hunk 1 above)
Comment (ghelmling) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Yes, this is no different from the other HBase version support (cdap-hbase-compat-0.96 gets installed even if you are running on HBase 0.98).  The startup scripts handle figuring out which version of the jars to include, the same as always, and are smart enough to flag the version of HBase 1.0 on CDH separately.

**Comment by awholegunch** [comment]
A few minor comments.

File: cdap-docs/admin-manual/source/installation/installation.rst (hunk 3 above)
Comment (ghelmling) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Yes, same as above.  Our approach so far has been to keep the docs simple and install all packages, then let the startup scripts figure out the right dependency.

File: cdap-docs/admin-manual/source/installation/installation.rst (hunk 2 above)
Comment (ghelmling) [inline cdap-docs/admin-manual/source/installation/installation.rst]: Done

**Comment by awholegunch** [comment]
LGTM.

**Pull request merged**